README.md

```diff
@@ -75,6 +75,33 @@ To set up a development environment for the CLI
 6. Start the in-cluster API replacement job (powered by [http://tilt.dev](tilt.dev)) with `make cluster-dev`
 7. make or make unit-tests to ensure everything built correctly.
 
+### Requirements/tools
+
+This is a list of the tools you may need to install:
+
+* [go](go.dev) -- Primary compiler for the CLI.
```

Comment:
GitHub gets confused when URLs look like relative URLs.

```suggestion
* [go](https://go.dev) -- Primary compiler for the CLI.
```

Reply:
fixed in 7857b906